### PR TITLE
DropDownButton/ButtonGroup height option change proceseed (T826558) 

### PR DESCRIPTION
**Pull request by ksercs**

### CODE REVIEW

File: js/ui/button_group.js [hunk 1]
```diff
@@ -268,7 +269,9 @@ const ButtonGroup = Widget.inherit({
             selectionRequired: false,
             onItemRendered: e => {
                 const width = this.option("width");
+                const height = this.option("height");
                 isDefined(width) && $(e.itemElement).addClass(BUTTON_GROUP_ITEM_HAS_WIDTH);
+                isDefined(height) && $(e.itemElement).addClass(BUTTON_GROUP_ITEM_HAS_HEIGHT);
```
Comment (DokaRus) [inline js/ui/button_group.js:274]: Add separate test for the ButtonGroup widget

Comment (ksercs) [reply js/ui/button_group.js]: Got it, thanks

Comment (ksercs) [reply js/ui/button_group.js:274]: Got it, thanks

File: testing/tests/DevExpress.ui.widgets/buttonGroup.tests.js [hunk 2]
```diff
@@ -81,6 +82,17 @@ QUnit.module("option changed", {
         assert.ok(buttons.eq(1).hasClass(BUTTON_GROUP_ITEM_HAS_WIDTH), "second item when button group has width");
     });
 
+    QUnit.test("change the height option", function(assert) {
```
Comment (DokaRus) [inline testing/tests/DevExpress.ui.widgets/buttonGroup.tests.js]: Check another cases:

1) Init widget with a specific height
```js
  dxButtonGroup({
    height: 500
  });
```

2) Change height to `null`

```js
  dxButtonGroup({
    height: 500
  });
  <...>
  buttonGroup.option("height", null)
  // $(`.${BUTTON_GROUP_ITEM_HAS_HEIGHT}`).length => 0 
```

**Review by DokaRus — APPROVED** [review]

**Review by DokaRus — APPROVED** [review]

**Review by jsdmitry — APPROVED** [review]

**Pull request merged**